### PR TITLE
WIP: Add AngularJS Material and make delete buttons use it for confirmation

### PR DESCRIPTION
**Pull request by wynnset**
- Change delete dialog confirmation button to explicitly state the action i.e. "Permanently remove this..."
- Change default action on delete dialoge to be "Cancel"

Note: Need to run `node_modules/gulp/bin/gulp.js` to install AngularJS Material and other requirements

### CODE REVIEW

**Comment by wynnset** [comment]
I tried a few different options and AngularJS Material is the one that works best for us. Here's the dialog option implemented. @lenglund could you please review and give me some feedback regarding both the wording and the look and feel?

You'll need to run node_modules/gulp/bin/gulp.js to install AngularJS Material and other requirements

Here are all the demos for AngularJS Material that we can utilise going forward: https://material.angularjs.org/latest/demo

File: bower.json [hunk 1]
```diff
@@ -44,7 +44,9 @@
     "angular-local-storage": "~0.5.2",
     "angular-promise-extras": "~0.1.8",
     "ng-kaltura-player": "~1.2.1",
-    "pdf.js-viewer": "~1.6.211"
+    "pdf.js-viewer": "~1.6.211",
+    "angular-material": "^1.1.6",
+    "angular-aria": "1.5.11"
```
Comment (andrew-gardener) [inline bower.json]: please also add `"angular-messages": "1.5.11"` (seems to be another new requirement)

**Comment by wynnset** [comment]
@andrew-gardener from my research, I found AngularJS Material and Angular UI Bootstrap (what we are using), which were compatible with AngularJS 1.5, but I was not able to find any bootstrap versions of AngularJS Material. Seems like no one has made that port (at least for AngularJS 1.5). Whatever I saw recommended upgrading from Bootstrap to MD (or trying and make them work side-by-side).

**Comment by andrew-gardener** [comment]
This one seems to be angular 1.5 & bootstrap 3 https://github.com/tilwinjoy/angular-bootstrap-material

**Comment by wynnset** [comment]
That one is not very polished and requires a lot of work to make things work. We might as well just stick with Angular UI if that's the case.

**Comment by lenglund** [comment]
I'd change the h2 to a p. Here's the wording I was thinking of:
message

Is it possible to highlight what will be deleted, so this is visible behind the confirmation?

Are we able to apply this to the exit confirmations too?

**Comment by wynnset** [comment]
@andrew-gardener if you think we don't need to add Angular Material / Angular Messages, then let's just go ahead and close this PR and delete the associated branch. I will start another branch to address #634, using uibModal instead. I'll wait for your response here.

**Pull request closed without merging**